internal/arith/generic.go

```diff
@@ -0,0 +1,66 @@
+// +build noasm arm64 arm
```

Comment:
I know it's currently the same for p751 and p503, but I think this should say `+build noasm !amd64`. Then you can still build this on your 32-bit machine, RISC-V or PowerPC.